src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -387,14 +398,31 @@ public static Calendar dateToCalendar(Date date) {
     /**
      * Returns the date object representing the next full hour from now.
      * Example: If now is 1055, this will return 1100
+     * @deprecated Use {@link TimeHelper#getNextHourInstant()} instead.
+     */
+    @Deprecated
+    public static Date getNextHourDate() {
+        return Date.from(getNextHourInstant());
+    }
+
+    /**
+     * Returns the java.time.Instant object representing the next full hour from now.
+     * Example: if now is 01/01/2016 1055, this will return an Instant representing 01/01/2016 1100.
+     * Wrapper for getNextHourFromInstant.
+     * @return
+     */
+    public static Instant getNextHourInstant() {
```

Comment:
an `Instant` does not have a concept of time in hour/minute. The result of this function should depend on a certain timezone, e.g. a timezone that has an offset of 01:30 will return a different result from UTC. Hence, I think this function needs to get a timezone, convert the `Instant` to a `ZonedDateTime`, modify the `ZonedDateTime` before converting it back to `Instant`.

Reply:
This method is actually quite redundant: it's used only for generating the feedback sessions form HTML, after which its value is ignored by the frontend anyway since it uses the browser time zone to select the correct option -
see https://github.com/TEAMMATES/teammates/pull/8332#discussion_r168262862. Maybe we can remove it altogether.

Reply:
Agree that this should be done in the front-end

Reply:
I've asked for it to be removed in that PR. On a side note, the current frontend UI (list of hours) may be ambiguous when nearing the transition period between DST and non-DST times, and is an issue we'll have to solve as part of #84.

Reply:
Okay, should I just replace all the instances of `TimeHelper.formatDate(TimeHelper.getNextHour())` with an empty string then?

Reply:
Can just avoid migrating it actually.

Reply:
Okay, then I'll revert this commit.

Reply:
Ohh I realised it's used in `HtmlHelper`; looks like we'll still need it.

Reply:
Wait - the HTML comparison part may not pick up the client value as it's made to the datepicker. Should be able to remove the method completely; can revert the commit first.